### PR TITLE
Test for issue #2056

### PR DESCRIPTION
**Pull request by mgperry**
See: hadley/dplyr#2056.

### CODE REVIEW

File: tests/testthat/test-binds.R [hunk 1]
```diff
@@ -105,6 +105,17 @@ test_that("bind_rows handles data frames with no columns (#1346)", {
   expect_equal(res$x, c(1, NA))
 })
 
+test_that("bind_rows handles lists with named NULL values (#2056)", {
+  skip("Currently failing")
+  df1 = data_frame(x=1, y=1)
```
Comment (krlmlr) [inline tests/testthat/test-binds.R]: Please use `<-` instead of `=`, for consistency with existing code.

Comment (mgperry) [inline tests/testthat/test-binds.R]: Done.

PS What is PTAL ?

File: tests/testthat/test-binds.R [hunk 2]
```diff
@@ -105,6 +105,17 @@ test_that("bind_rows handles data frames with no columns (#1346)", {
   expect_equal(res$x, c(1, NA))
 })
 
+test_that("bind_rows handles lists with named NULL values (#2056)", {
+  skip("Currently failing")
+  df1 <- data_frame(x=1, y=1)
```
Comment (hadley) [inline tests/testthat/test-binds.R]: I think you could simplify this still further:

```R
df1 <- data_frame(x = 1, y = 1)
df2 <- data_frame(x = 2, y = 2)

lst1 <- list(df1, NULL, df2)
lst2 <- list(df1, df2)
```

Comment (mgperry) [inline tests/testthat/test-binds.R]: Yes, df2 is never used! Will edit.

File: tests/testthat/test-binds.R [hunk 3]
```diff
@@ -105,6 +105,16 @@ test_that("bind_rows handles data frames with no columns (#1346)", {
   expect_equal(res$x, c(1, NA))
 })
 
+test_that("bind_rows handles lists with NULL values (#2056)", {
+  skip("Currently failing")
+  df1 <- data_frame(x=1, y=1)
```
Comment (krlmlr) [inline tests/testthat/test-binds.R]: Could you please add spaces around `=`, to match surrounding style?

File: tests/testthat/test-binds.R [hunk 4]
```diff
@@ -105,6 +105,16 @@ test_that("bind_rows handles data frames with no columns (#1346)", {
...
+  df2 <- data_frame(x=2, y=2)
+  lst1 <- list(a=df1, NULL, b=df2)
+  lst2 <- list(a=df1, b=df2)
+
+  expect_equal(bind_rows(lst1, .id="bind_rows_id"), bind_rows(lst2, .id="bind_rows_id"))
```
Comment (krlmlr) [inline tests/testthat/test-binds.R]: It would be a tad safer if you compared against known results, e.g., `df <- data_frame(x = c(1, 2), y = c(1, 2))`

**Comment by mgperry** [comment]
Done & done, nearly forgot the "<-" again...

**Comment by krlmlr** [comment]
Thanks. Please double-check spaces around "=" and indent width.

**Comment by mgperry** [comment]
Should be fine now. Have you guys considered publishing a style guide for contributed code?

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**